Cargo.lock

```diff
@@ -269,7 +269,7 @@ dependencies = [
 [[package]]
 name = "linux-loader"
 version = "0.1.0"
-source = "git+https://github.com/bjzhjing/linux-loader#912dbb58c8a859c10735066d5a2089cd4073904c"
+source = "git+https://github.com/bjzhjing/linux-loader#6f60affdcec9be0b3768587b19854d39a4509f4c"
```

Comment:
Please drop this from your changes.

Reply:
Ok but the build won't pass. Are you planning to update it separately?

Reply:
I meant it should be a separate commit.